www/docs/customization/announce.md

```diff
@@ -30,5 +30,30 @@ announce:
     message_template: 'Awesome project {{.Tag}} is out!'
 ```
 
+## Slack
+
+For it to work, you'll need to [create a new Incoming Webhook](https://api.slack.com/messaging/webhooks), and set some environment variables on your pipeline:
+
+- `SLACK_WEBHOOK`
+- `SLACK_CHANNEL`
+- `SLACK_USERNAME`
+- `SLACK_ICON_EMOJI`
+- `SLACK_ICON_URL`
+
+Then, you can add something like the following to your `.goreleaser.yml` config:
+
+```yaml
+# .goreleaser.yml
+announce:
+  slack:
+    # Wether its enabled or not.
+    # Defaults to false.
+    enabled: true
+
+    # Message template to use while publishing.
+    # Defaults to `{{ .ProjectName }} {{ .Tag }} is out! Check it out at {{ .GitURL }}/releases/tag/{{ .Tag }}`
+    message_template: 'Awesome project {{.Tag}} is out!'
+```
+
```

Comment:
docs are outdated now, aren't they? we changed most props to be on the config file instead of yaml...

Reply:
Oh, I forgot to update docs. Updated. 👍